web/src/app/core/services/monitor.service.ts

```diff
@@ -0,0 +1,89 @@
+// Core components
+import { Injectable } from '@angular/core';
+import { MatSnackBar } from '@angular/material';
+import { Router } from '@angular/router';
+
+// Firestore modules
+import { AngularFirestore } from '@angular/fire/firestore';
+import * as firebase from 'firebase';
+
+// Rxjs operators
+import { Observable } from 'rxjs';
+import { switchMap } from 'rxjs/operators';
+
+// Dashboard hub models and services
+import { MonitorModel, ProjectModel } from '../../shared/models/index.model';
+import { ActivityService } from './activity.service';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class MonitorService {
+
+  constructor(
+    private afs: AngularFirestore,
+    private activityService: ActivityService,
+    private snackBar: MatSnackBar,
+    private router: Router
+  ) {
+  }
+
+  /**
+   *
+   * @param monitors This function will return the monitor by id
```

Comment:
I think the method description should be above the parameters not with the first parameter, the text after the parameter is usually the parameter description

Reply:
right